### PR TITLE
much more detail glossary

### PR DESCRIPTION
**Pull request by tpscrpt**
I originally wrote most of these and am not very proud of what's contained here. 
Made some effort to further the detail on the initial few definitions and am looking to complete this soon.

Will push more work to this branch and make another PR if it gets screwed regardless.

TY for your time :beers:

### CODE REVIEW

**Review by kristjank — APPROVED** [review]

File: docs/other/glossary.md [hunk 1]
```diff
@@ -6,21 +6,52 @@ title: "Glossary"
...
+Once an account is recorded as an eligible delegate in the Blockchain, it may be the recipient of vote transactions (pledges) by other accounts.
+
+For Ark, the top 51 delegates are responsible of forging blocks and maintaining consensus over the transaction history of the Blockchain. If a deleagte is inactive for a given 51 block round, it temporarily loses its ability to forge a block and is replaced by another delegate.
+
+Delegated block production is an advantage for a Blockchain. It allows for a seamless processing of blocks because the delegates are incentivized, through monetary reward, to maintain their voters' pledges by acting appropriately.
 
 #### What is a "Loader"?
 A loader syncs the peer with the network history.
 
 #### What is a "Peer"?
```
Comment (boldninja) [inline docs/other/glossary.md]: That is wrong. Peers in ARK are not normal users, but those that are running full nodes. Also having more peers in an unified network of peers doesn't mean data gets to destination faster.

**Comment by tpscrpt** [comment]
Thanks, was confusing to me considering the presence of both peer and node
definition

**Comment by kristjank** [comment]
@JeremiGendron can you please resolve and update requested changes. tnx

cc @mvega3

**Comment by tpscrpt** [comment]
On it

**Comment by tpscrpt** [comment]
View new PR #33

**Pull request closed without merging**